ipalib/constants.py

```diff
@@ -171,6 +171,9 @@
     ('ca_install_port', None),
     ('ca_agent_install_port', None),
     ('ca_ee_install_port', None),
+    # How long http connection should wait before trying again [seconds].
+    # Do not mistake with "startup_timeout"
+    ('ca_status_http_timeout', 30),
```

Comment:
I don't like adding this to api.env, if you really have to, can't you make this more general?